[2021-12-13 14:34:42,591] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse 2021-12-13T14:34:40.356629+00:00 [queued]>
[2021-12-13 14:34:42,606] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: cleansing_dag.cleanse 2021-12-13T14:34:40.356629+00:00 [queued]>
[2021-12-13 14:34:42,607] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 14:34:42,607] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-12-13 14:34:42,607] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 14:34:42,621] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): cleanse> on 2021-12-13T14:34:40.356629+00:00
[2021-12-13 14:34:42,625] {standard_task_runner.py:52} INFO - Started process 56 to run task
[2021-12-13 14:34:42,628] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'cleansing_dag', 'cleanse', '2021-12-13T14:34:40.356629+00:00', '--job-id', '12', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/kym_cleansing.py', '--cfg-path', '/tmp/tmpkcg8p8k_', '--error-file', '/tmp/tmpfi7svnla']
[2021-12-13 14:34:42,629] {standard_task_runner.py:77} INFO - Job 12: Subtask cleanse
[2021-12-13 14:34:42,666] {logging_mixin.py:104} INFO - Running <TaskInstance: cleansing_dag.cleanse 2021-12-13T14:34:40.356629+00:00 [running]> on host c2664a5050de
[2021-12-13 14:34:42,700] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=cleansing_dag
AIRFLOW_CTX_TASK_ID=cleanse
AIRFLOW_CTX_EXECUTION_DATE=2021-12-13T14:34:40.356629+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-13T14:34:40.356629+00:00
[2021-12-13 14:39:13,783] {python.py:118} INFO - Done. Returned value was: None
[2021-12-13 14:39:13,899] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=cleansing_dag, task_id=cleanse, execution_date=20211213T143440, start_date=20211213T143442, end_date=20211213T143913
[2021-12-13 14:39:14,070] {local_task_job.py:146} INFO - Task exited with return code 1
